lua/lspconfig/server_configurations/zk.lua

```diff
@@ -1,5 +1,13 @@
 local util = require 'lspconfig.util'
 
+local function find_zk_ancestor(startpath)
+  return util.search_ancestors(startpath, function(path)
```

Comment:
could you use vim.fs moudle replace util module here. It's helpful when we want remove some fs util functions in util.lua in future.

Reply:
Updated.